pages/validator/external-chains/filecoin.mdx

```diff
@@ -117,6 +117,13 @@ Modify the lotus config file `~/.lotus/config.toml`. First, enable the Eth RPC n
   # type: bool
   # env var: LOTUS_FEVM_ENABLEETHRPC
   EnableEthRPC = true
+
+[Libp2p]
+  ListenAddresses = ["/ip4/<your-public-ip>/tcp/<port-with-firewall-allow>"]
```

Comment:
Wanna add an example value so the formatting is easy to understand?

Reply:
sure